packages/client/README.md

```diff
@@ -27,21 +41,41 @@ We invite you to explore and would be delighted if you give us feedback on your
 
 ## Installation
 
-To obtain the latest version, simply require the project using `npm`:
+To be able to run the EthereumJS client, you need a working [Node.js](https://nodejs.org/en/) installation, see e.g. these [docs](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) from the npm documentation for combined instructions on how to install Node.js and eventually npm.
+
+We currently recommend to run the client with a recent Node.js version `18` installation.
+
+### NPM Installation
+
+Client releases are done on a regular basis on [npm](https://www.npmjs.com/package/@ethereumjs/client).
```

Comment:
Can we define what "regular basis" means?  I might drop this since I would say we do more periodic/occasional updates than "regular".  A nit but just don't want to overcommit us.